pymchelper/input_output.py

```diff
@@ -20,7 +20,15 @@
 
 
 class AggregationType(IntEnum):
-    """Enum for different types of aggregation."""
+    """
+    Enum for different types of aggregation.
+    Below few examples of such aggregation types used in SHIELD-HIT12A:
+      - NoAggregation is used for density (RHO) and material scorer.
+      - Sum is used for particle counter (COUNT).
+      - AveragingCumulative is used for dose and fluence scorers.
+      - AveragingPerPrimary is used for LET scorers (TLET and DLET).
```

Comment:
X = (sum^j_i x_i  * n_i) / sum^j_i n_i)    # meaning x_i is already an average so we need a weighted average over the all jobs